gdnative-core/src/macros.rs

```diff
@@ -134,7 +134,7 @@ macro_rules! godot_nativescript_init {
             }
 
             let __result = ::std::panic::catch_unwind(|| {
-                $callback($crate::init::InitHandle::new(handle));
+                $callback($crate::nativescript::init::InitHandle::new(handle));
```

Comment:
Maybe the NativeScript-related macros should be behind the feature flag too, if that's possible?

Reply:
I was thinking about putting them in a separate `nativescript/macros.rs` file too, but I wasn't sure if that is needed. I can just put a `#[cfg(feature = "nativescript")]` too, whatever you prefer!

Reply:
I put the macros only used for NativeScript interactions into a separate module in `nativescript`, hope that's okay?

Reply:
Yeah, that looks good!